index.js

```diff
@@ -0,0 +1,53 @@
+//Psuedocode
+// 
+
+
+// This is the flipCoin function in JS
+function flipCoin () {
+        // assign random value where 0 = heads & 1 = tails
+        var flip = Math.floor(Math.random() * 2)
```

Comment:
🔵 Don't forget! `var` is the old way to declare variables. You should be using `const` if the value won't change or `let` if it will.